### PR TITLE
Various input fixes

### PR DESCRIPTION
**Pull request by bastianallgeier**
This PR is coming from the inputbox refactoring and moves over a couple of the fixes

## Fixes
- Fixed translation string for the blocks field
- The link field shows up correctly if no options are defined

### CODE REVIEW

File: panel/src/components/Forms/Field/BlocksField.vue [hunk 1]
```diff
@@ -108,14 +108,14 @@ export default {
 			return this.$helper.object.length(this.fieldsets) > 0;
 		},
 		isEmpty() {
-			return this.value.length === 0;
+			return this.value?.length === 0;
```
Comment (distantnative) [inline panel/src/components/Forms/Field/BlocksField.vue]: Why is this needed? Like just curious to understand. Value has the array type and a default set. Why does it result in being null?

Comment (bastianallgeier) [reply panel/src/components/Forms/Field/BlocksField.vue]: Thinking some more about it, I probably caused this unintentionally in the UI lab. The value is null there by default for all examples to achieve proper reactivity. But this is not really representative of real-world usage. It just simplifies the example setup. I will check this again.

Comment (bastianallgeier) [reply panel/src/components/Forms/Field/BlocksField.vue]: Ok, it's a problem how the default value is set in the examples. I've reverted the changes and fixed the examples instead.

**Review by distantnative — APPROVED** [review]
